JAMClock.py

```diff
@@ -19,7 +19,8 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 
-import os, gtk, pygtk, gobject, sys, socket, pygame
+from gi.repository import Gtk
+import os, pygtk, gobject, sys, socket, pygame
```

Comment:
You should also do `from gi.repository import GObject`. As it is, you import `gobject` but then reference `GObject`. It shouldn't work.

Reply:
Yikes! My bad. I will make the changes immediately.